docs/security_conop/Evaluation_Artifacts.rst

```diff
@@ -14,5 +14,6 @@ our :ref:`help-public-resources` or enter a PR against the `simp-doc`_ project.
 
    evaluation_artifacts/SCAP
    evaluation_artifacts/InSpec
+   evaluation_artifacts/RPM_Inspection
```

Comment:
This should fall under either an SCAP or InSpec (or both) scan output section.

It may make sense to keep this as an external include file (`.inc`) and have both reports include that file.